datacats/userprofile.py

```diff
@@ -86,13 +113,13 @@ def create(self, project, target_name, stream_output=None):
             web_command(
                 command=[
                     "ssh", _project_user_host(project), "create", target_name,
-                    ],
+                ],
```

Comment:
hey, why are you moving my brackets and parens? either style is fine according to pep8, I tend to line them up with the indent.